chocolatey-community/chocolatey-community-validation
# (doc) Create release procedure issue template

## Description Of Changes

This creates a new issue template that can be used when we are preparing to start releasing a new version of the project.

## Motivation and Context

To have a set item of steps to follow when releasing the product

## Testing

This was tested in my own fork by doing

1. Change the default branch to `procedure-template`.
2. Attempt to create a new issue
3. Verify that the information prepoulated in the issue had the correct information.

### Operating Systems Testing

N/A

## Change Types Made



## Change Checklist


## Related Issue

N/A
